Paralayout/FixedDistributionProxy.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright © 2025 Nick Entin
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+@MainActor
+public final class FixedDistributionProxy: ViewDistributionSpecifying {
+
+    // MARK: Initialization
+
+    /// Creates a fixed distribution proxy of the specified length.
+    ///
+    /// When applying a distribution, this distribution proxy will be assigned a rect with the specified `length` along the axis of distribution.
+    public init(length: CGFloat) {
+        self.length = length
+    }
+
+    // MARK: Public
+
+    /// The length along the axis of distribution this distribution proxy should be apportioned.
+    public let length: CGFloat
```

Comment:
naming things is hard. when I see `length`, I wonder "what is the `width`?"

maybe `dimension`? Or better yet: `fixed` to align with the `ViewDistributionItem.fixed`?

Reply:
Naming things is _very_ hard.

`length` is the most generic term I could think of for "measurement of one dimension of a size." `FixedDistributionProxy(length: 24)`

`dimension` feels to me like I'm supposed to pass in _which_ dimension (like `.width` or `.height`), not a length along the dimension. `FixedDistributionProxy(dimension: 24)`

`fixed` just reads really awkwardly imo - `FixedDistributionProxy(fixed: 24)`

Could go super generic and just say "value" or similar - `FixedDistributionProxy(value: 24)` - but I think that's even less clear it's talking about a pt measurement.

Reply:
`fixedLength`? I don't love it. I just yearn for parallelism.

maybe what I'd want is `DistributionProxy(fixed: CGFloat)` and then `DistributionProxy(flexible: CGFloat)`?

but what you have works for now. I just want to keep iterating